### PR TITLE
map fix for cryo

### PR DESCRIPTION
**Pull request by Intigracy**
Code still sucks and needs fixing so this isn't neccessary, but now all the cryo units have a gas pump after the oxygen canisters so you can manipulate the pressure in the tubes.

### CODE REVIEW

**Comment by PJB3005** [comment]
The ones on deff and box paramedic are broken and cause the actual cryo tube network to not be able to reach the purge pump.

**Comment by ghost** [comment]
Fix it properly.

**Comment by Gringonius** [comment]
What the Englishman said.

**Comment by kol1th** [comment]
> so this isn't neccessary

agreed

**Comment by Aurx** [comment]
I am of the personal opinion that cryo being able to be overpressurized is a feature that should be retained.

**Comment by Exxion** [comment]
As you can clearly see by emoji, I'm in agreement with Aurx

**Comment by SonixApache** [comment]
better this than everyone getting crushed in cryo due to 'bugfixes'

**Comment by Kammerjunk** [comment]
I don't like it. I like it more than cryo properly set up hurting people, but I don't like it. I don't think it's a good change if the issue is that cryo is "broken." Admittedly, I only play medbay just less than once a day at average.
Any word on the desired pressure? I assume it would be 101.3 kPa, but how should I know, I very rarely do atmos. Will this be written anywhere or will it be another thing you just have to figure out by playing?

**Comment by Exxion** [comment]
Your assumption would be correct, assuming cryo atmos works as it should

**Comment by Intigracy** [comment]
You can still overpressurize it by turning the pump up.

Cryo currently deals damage to you while you're inside of it, you guys are
whining about a bugfix after people who actually play medbay were whining
all day about wanting it fixed, holy shit.

**Comment by Exxion** [comment]
Does the pump start on and set to the right pressure?

**Comment by Intigracy** [comment]
You can see from the images that they start on.

It's default pressure which is 101.13 or whatever it is, which is the
pressure of Atmos everywhere.

**Comment by Exxion** [comment]
> Implying images from the mapper reliably have any bearing on the states of the items pictured

Anyway I doubt anyone thinks this is worse than what we have right now

**Comment by Kammerjunk** [comment]
Yeah, gas pumps are at 101.3 kPa by default, I think. As long as it's just another thing to toggle on, whatever.

**Comment by Intigracy** [comment]
Don't even need to toggle it.

File: maps/defficiency.dmm [hunk 1]
```diff
@@ -698,7 +698,7 @@
 "anv" = (/obj/structure/closet/secure_closet/engineering_atmos,/obj/machinery/atmospherics/pipe/simple/supply/hidden{dir = 4},/turf/simulated/floor,/area/engineering/atmos)
 "anw" = (/obj/structure/window/reinforced{dir = 8},/obj/structure/window/reinforced{dir = 4},/obj/structure/grille,/obj/structure/window/reinforced{dir = 1},/obj/machinery/atmospherics/pipe/simple/supply/hidden{dir = 4},/turf/simulated/floor/plating,/area/engineering/atmos)
 "anx" = (/obj/machinery/atmospherics/pipe/simple/scrubbers/visible,/obj/machinery/atmospherics/pipe/simple/supply/hidden{dir = 4},/turf/simulated/floor,/area/engineering/atmos)
-"any" = (/obj/machinery/atmospherics/binary/volume_pump{dir = 2; name = "Filter to Waste"},/obj/machinery/atmospherics/pipe/simple/supply/hidden{dir = 4},/turf/simulated/floor,/area/engineering/atmos)
+"any" = (/obj/machinery/atmospherics/binary/volume_pump{dir = 2; name = "Emergency Vacuum Pump"},/obj/machinery/atmospherics/pipe/simple/supply/hidden{dir = 4},/turf/simulated/floor,/area/engineering/atmos)
```
Comment (Exxion) [inline maps/defficiency.dmm:701]: UNATOMIC

But it's already merged. Fuck.

Comment (Intigracy) [reply maps/defficiency.dmm:701]: Oh, I accidentally edited state instead of the specific one.

My bad.

Comment (Exxion) [reply maps/defficiency.dmm:701]: FIX IT

**Comment by ghost** [comment]
> MERGED

**Pull request merged**